pkg/cli/common.go

```diff
@@ -46,8 +46,8 @@ type NameSpaceResults struct {
 	UTS           string
 }
 
-// BudResults represents the results for Bud flags
-type BudResults struct {
+// BuildResults represents the results for Build flags
+type BuildResults struct {
```

Comment:
Leave the structs the same or you will cause other packages to rebuild and should bump buildah main version.